contributors/design-proposals/scheduling/cluster-scoped-resources.md

```diff
@@ -0,0 +1,76 @@
+# Cluster Scoped Resources
+
+## Abstract
+Cluster scoped resources are consumable resources that do not belong to any specific node but instead are available across mulitple nodes in a cluster. These resources are accounted as other consumable resources and should be usable by the scheduler while deciding if a pod can actually be scheduled.
+
+
+## Motivation
+Resources in Kubernetes such as cpu and memory are available at a node level and can be consumed by pods by requesting them. However there are some resources that do not belong a specific node, but they are consumable across all or a group of nodes in the cluster. As an example, IP addresses in a pool can be shared across pods running on multiple nodes in a network scope. Another use case could be, locally attached shared storage in a rack, which is consumable across several nodes. Hence there is a need to represent such a resource at cluster level which is consumable acroass all or a group of nodes in the cluster.
```

Comment:
This is more like a node _group_ scoped resources in the examples.

Reply:
Please add a list of 5-8 example resources that would be tracked like this.  I’d like more validation and concrete discussion on each type to guide design.

Reply:
+1. There are many use cases for extending resource APIs and I'd like to first get a collection of use-cases before identifying possible solutions.

Reply:
Added few use cases